src/main/resources/public/js/getAndAddContracts.js

```diff
@@ -245,7 +245,7 @@ var projectContractsCount = -1;
                         //adding contracts to table
                         callback({ data: contracts.map(contractAsTableRowArray) });
                         $('[data-toggle="tooltip"]').tooltip();
-                        $(".contractAgenda").each(
+                        $("#contracts .contractAgenda").each(
```

Comment:
Added  table  id  to selector, to avoid conflicts if there are other elements with the same class attached.